x-pack/plugins/enterprise_search/public/applications/app_search/components/not_found/not_found.tsx

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { APP_SEARCH_PLUGIN } from '../../../../../common/constants';
+import { PageTemplateProps } from '../../../shared/layout';
+import { NotFoundPrompt } from '../../../shared/not_found';
+import { AppSearchLogo } from '../../../shared/not_found/logos';
+import { SendAppSearchTelemetry } from '../../../shared/telemetry';
+import { AppSearchPageTemplate } from '../layout';
+
+export const NotFound: React.FC<PageTemplateProps> = ({ pageChrome = [] }) => {
+  return (
+    <AppSearchPageTemplate pageChrome={[...pageChrome, '404']} template="centeredContent">
```

Comment:
So, I checked the translation JSON files and it looks like those translations _aren't_ translating numbers / left "404" in current messages as is in multiple plugins, so I _think_ I'm okay to leave this untranslated - but if y'all feel strongly I can change this, just let me know

Reply:
This makes sense, HTTP codes use ["Arabic" numerals ](https://en.wikipedia.org/wiki/Arabic_numerals) exclusively

Reply:
Ooo nice find! Thanks Byron!